projects/packages/publicize/src/rest-api/class-connections-controller.php

```diff
@@ -64,6 +65,58 @@ public function register_routes() {
 						),
 					),
 				),
+				array(
+					'methods'             => WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'create_item' ),
+					'permission_callback' => array( $this, 'create_item_permissions_check' ),
+					'args'                => array(
+						'keyring_connection_ID' => array(
+							'description' => __( 'Keyring connection ID.', 'jetpack-publicize-pkg' ),
+							'type'        => 'integer',
+							'required'    => true,
+						),
+						'external_user_ID'      => array(
+							'description' => __( 'External User Id - in case of services like Facebook.', 'jetpack-publicize-pkg' ),
+							'type'        => 'string',
+						),
+						'shared'                => array(
+							'description' => __( 'Whether the connection is shared with other users.', 'jetpack-publicize-pkg' ),
+							'type'        => 'boolean',
+						),
+					),
+				),
+				'schema' => array( $this, 'get_public_item_schema' ),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<connection_id>[0-9]+)',
+			array(
+				'args'   => array(
+					'connection_id' => array(
+						'description' => __( 'Unique identifier for the connection.', 'jetpack-publicize-pkg' ),
+						'type'        => 'string',
+						'required'    => true,
+					),
+				),
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
```

Comment:
I think the problem is that EDITABLE contains POST as a method (https://github.com/WordPress/wordpress-develop/blob/6.7/src/wp-includes/rest-api/class-wp-rest-server.php#L40). 

Perhaps we should make the connection ID parameter optional, then decide whether to create or update the connection based on that?

Reply:
Ignore me, I thought it was because we had two routes registered and one was stamping on the other.